setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+import os
+
+setup(name='codalab-cli',
+    version='0.2.1',
```

Comment:
Ok, I did some digging, what Django does is they store the version number in [django/__init__.py](https://github.com/django/django/blob/master/django/__init__.py) (like we do) and then reads that number from [setup.py](https://github.com/django/django/blob/master/setup.py#L32). If that sounds good to you, I'll go ahead and make that change.